### ISSUE TITLE
Give the option to only preload data

### ISSUE THREAD

**Issue by arthurcavallari** [pawelczak/EasyAutocomplete#168]
It'd be handy give the option to pre-load the source data only once as opposed of hitting the endpoint on every KeyUp.

At the moment I'm getting around this with the following code:

$.get("/resource/list.json")
    .done(function(data) {
      initAutoComplete(data);
    });

  function initAutoComplete(autocompleteDataSource) {
    var options = {
      data: autocompleteDataSource,
      //url: clientListURL,  
      template: {
        type: "custom",
        method: function(value, item) {
          return "<div data-channel='" + item.channel + "' data-url='" + item.url + "' >" + value + "</div>";
        }
      },

      getValue: "name",
      list: {
        match: {
          enabled: true
        },
      }
    };
    $('#autoCompleteField').easyAutocomplete(options);
  }

**Comment by pawelczak** [comment]
Yes, you are right. This feature is much needed, I hope it'll get introduced in near future. I really like your hack, that resolves this issue.

Thanks for your suggestion !
